src/mudlet.cpp

```diff
@@ -534,6 +534,11 @@ mudlet::mudlet()
         if (!host) {
             return;
         }
+        host->mpEditorDialog = createMudletEditor();
+        if (!host->mpEditorDialog) {
+            return;
```

Comment:
Not really, I was expecting `createMudletEditor()` to create the editor - and to moan if it failed - I've looked into it and if that one does not manage to do the creation I thing protesting a failure to create the editor is not unreasonable...

BTW Would it be useful to convert `(dlgTriggerEditor*) Host::mpEditorDialog` to a `(QPointer<dlgTriggerEditor>) Host::mpEditorDialog` to ensure the pointer gets reset if the editor is **delete**d?

Reply:
> > No because it's normal not to have the editor window open when you're playing
> 
> Actually I do, all the time. The editor likes to reset to the very top when you reopen it instead of scrolling to where you left off. When editing scripts with 1900 lines this gets taxing!

This has been a long standing wishlist request in #1894 
Feel free to weigh in there for more details! :+1: